Content.Server/Movement/Components/FootstepModifierComponent.cs

```diff
@@ -20,7 +20,7 @@ public class FootstepModifierComponent : Component
 
         public void PlayFootstep()
         {
-            SoundSystem.Play(Filter.Pvs(Owner), SoundCollection.GetSound(), Owner.Transform.Coordinates, AudioParams.Default.WithVolume(-2f));
```

Comment:
Honestly, I don't think it makes sense at all that footstep sounds follow an entity. Revert the change